h2o-core/src/main/java/water/api/ParseHandler.java

```diff
@@ -26,20 +22,32 @@ public ParseV3 parse(int version, ParseV3 parse) {
                                       parse.domains, parse.na_strings,
                                       null,
                                       new ParseWriter.ParseErr[0], parse.chunk_size,
-                                      parse.decrypt_tool != null ? parse.decrypt_tool.key() : null);
+                                      parse.decrypt_tool != null ? parse.decrypt_tool.key() : null, parse.skipped_columns);
 
     if (parse.source_frames == null) throw new H2OIllegalArgumentException("Data for Frame '" + parse.destination_frame.name + "' is not available. Please check that the path is valid (for all H2O nodes).'");
     Key[] srcs = new Key[parse.source_frames.length];
     for (int i = 0; i < parse.source_frames.length; i++)
       srcs[i] = parse.source_frames[i].key();
 
+    if ((setup.getParseType().name().toLowerCase().equals("svmlight") ||
+            (setup.getParseType().name().toLowerCase().equals("avro") ))
```

Comment:
That is a good idea.  Yes, I believe so.  Let me find it and fix it.

Reply:
Nope, I checked.  OrcParser has OrcParseInfo but not sure about the other ones.  We only go through the parseHandler a few times before the parse.  It should not cost too much time.

I am afraid that people like to change parser names, if they do, this will break.....